circle.yml

```diff
@@ -1,3 +1,7 @@
+machine:
+  php:
+    version: 5.6.14
```

Comment:
Hmm... yeah I think you're right actually – only seeing it mentioning ubuntu now looking at it.

I'll try pull it from homebrew in that case.